db/migrate/20150612104023_add_gov_response_at_to_petitions.rb

```diff
@@ -0,0 +1,5 @@
+class AddGovResponseAtToPetitions < ActiveRecord::Migration
+  def change
+    add_column :petitions, :gov_response_at, :timestamp
```

Comment:
Not sure about the name - `government_response_at` or `official_response_at` are alternatives. Ask @h-lame what he thinks.